node/src/command.rs

```diff
@@ -1,11 +1,9 @@
 use crate::{
     chain_spec,
-    cli::{
-        Cli,
-        RelayChainCli,
-        Subcommand,
-    },
+    cli::{Cli, RelayChainCli, Subcommand},
+    service::{new_partial, DataHighwayRuntimeExecutor}
 };
+use datahighway_runtime::RuntimeApi;
```

Comment:
suggest combine this line `use datahighway_runtime::RuntimeApi;` and `use datahighway_runtime::Block;` into :
`use datahighway_runtime::{Block, RuntimeApi};`, and put it on the line after `use log::info;`, like here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/node/src/command.rs#L10

Reply:
added to https://github.com/DataHighway-DHX/node/issues/231